service/src/main/kotlin/io/provenance/explorer/service/async/AsyncService.kt

```diff
@@ -122,4 +125,10 @@ class AsyncService(
         logger.info("Updating Gas volume")
         TxGasCacheRecord.updateGasFeeVolume()
     }
+
+    @Scheduled(cron = "0 0 0/1 * * ?") // At the start of every hour
```

Comment:
Depending on how fast or slow this is, we may want to update this every 12 hours - 1 day.

Reply:
changed to once a day at 1am.